<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiIcon</class>
 <widget class="QWidget" name="UiIcon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UiIcon</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>301</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid lightblue;border-radius: 15px;
background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_usbpicture">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border: 5px solid lightblue;border-radius: 10px;
selection-background-color: rgb(255, 170, 0);
QProgressBar::chunk {background-color: rgb(202, 165, 14);}
</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_waring">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>120</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;
font: 15px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_usbpicture</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_waring</zorder>
  <zorder>listView</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
